aptos-move/framework/aptos-framework/sources/aptos_coin.move

```diff
@@ -8,7 +8,12 @@ module aptos_framework::aptos_coin {
     use std::option::{Self, Option};
 
     use aptos_framework::coin::{Self, BurnCapability, MintCapability};
+    use aptos_framework::fungible_asset::{generate_mint_ref, generate_transfer_ref, generate_burn_ref};
+    use aptos_framework::object;
+    use aptos_framework::primary_fungible_store;
     use aptos_framework::system_addresses;
+    use aptos_framework::system_addresses::assert_aptos_framework;
```

Comment:
like this... also note you can merge this with the previous {Self, ...}

Reply:
it's draft. fixed now.